tasty-plutus/src/Test/Tasty/Plutus/Internal/Context.hs

```diff
@@ -124,14 +149,19 @@ data Input
 -}
 data Output
   = -- | @since 3.0
-    Output ExternalType Value
+    Output ExternalType ValueType
   deriving stock
     ( -- | @since 1.0
       Show
     )
 
-{- | A minting result. Do not use this for tokens being minted by the current
- minting policy; pass those as 'Tokens' to the test instead.
+{- | 'Value' minted with minting policy other than the tested one.
```

Comment:
This can be worded a bit better. Something like:

A 'Value' minted with a minting policy other than the one being tested.